app/src/main/java/example/chatea/servicecamera/CameraService.java

```diff
@@ -24,6 +24,7 @@
 
     private Camera mCamera;
     private MediaRecorder mMediaRecorder;
+    public static String DataPath;
```

Comment:
Please, do **NOT** use static variable to pass your data...
Use Application class to do so.

btw, don't naming variable start with upper case.